app/src/main/res/values/settings_strings.xml

```diff
@@ -67,9 +67,9 @@
     <string name="settings_item_logout_confirm_title">Are you sure you want to log out?</string>
 
     <!--Recovery Kit-->
-    <string name="recovery_title">Recovery kit</string>
-    <string name="recovery_text">Your recovery kit</string>
-    <string name="recovery_sub_text">If you switch devices, you can easily restore your wallet. No private keys needed.</string>
+    <string name="recovery_title">Security &amp; privacy</string>
```

Comment:
can you fix and remove `t` in values/settings_strings.xml:87? seed_phrase_hint_2